tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs

```diff
@@ -22,6 +22,8 @@ public class RemoveInternetSourcesFromNuGetConfig : Task
         [Required]
         public string NuGetConfigFile { get; set; }
 
+        public bool OfflineBuild { get; set; }
```

Comment:
The task comment looks like it's describing something else--remove it? Maybe add a comment on this property to explain what it's doing in particular?

Reply:
Fixed the task comment and added a comment here.